pkg/trace/sampler/remote_config_handler.go

```diff
@@ -0,0 +1,58 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package sampler
+
+import (
+	"time"
+
```

Comment:
I can't just add the following two imports like that I suppose.

But I didn't really understand what I'm allowed to import or not in `pkg/trace`